performance/collector/es.py

```diff
@@ -0,0 +1,89 @@
+
+import json
+import requests
```

Comment:
This violates pep8 in many ways: 

- You should have 3 groups of libraries separated by blank line in alphabetical order (python libs, thrid party libs, your modules) 
- Always import only modules 

Resulting list should look like: 

```
import datetime 
import json 
import os

import requests
import six

from rally import api
from rally.common import logging 
from rally import exceptions 
from rally.task import exporter 
```